Assignments/Object oriented/HW1.js

```diff
@@ -30,24 +30,51 @@ class Product {
 		this.name = name;
 		this.price = price;
 	}
+}
+
+class Inventory {
+	constructor () {
+
+	}
 
-	save () {
+	addProduct(newProduct){
+		//adds the product to sparkfun
 		$.ajax({
 			url: 'http://data.sparkfun.com/input/q5nngNQvJ0uJWYMLL03V',
 			headers: {
 				'Phant-Private-Key':'BVRRjJXlBWIPqJ6zzWng'
 			},
 			method: 'POST',
 			dataType: 'json',
-			data: {name: this.name, tags: this.tags, stores: this.stores, price: this.price},
+			data: {name: newProduct.name, tags: newProduct.tags, stores: newProduct.stores, price: newProduct.price},
 			success: function(data) {
 				console.log('success: ' + data);
 				alert("Saved!");
 			}
 		})
 	}
+
+	showAllProducts() {
+		//shows all of the products from sparkfun
+		$.ajax({
+			type: "GET",
+			url: "https://data.sparkfun.com/output/q5nngNQvJ0uJWYMLL03V.json",
+			success: (response) => {
+				console.log(response);
+				$("div").html("<table style='width:500px'><tr><th>Name</th><th>Price</th><th>Tags</th><th>Stores</th></tr></table>");
+				for(let z=0;z<response.length;z++) {
```

Comment:
```js
for (const obj of response) {
  console.log(obj.name);
}
```
If you didn't need to use the `z` anywhere this will return just the object itself. You can also combine strings using this new ES6 syntax.

```js
$("table").append(`<tr><td>${response[z].name}</td><td>$${response[z].price}</td><td>${response[z].tags}</td><td>${response[z].stores}</td></tr>");
```